app/src/main/java/com/wabadaba/dziennik/ui/timetable/TimetableFragment.kt

```diff
@@ -54,20 +55,24 @@ class TimetableFragment : LifecycleFragment() {
                 fragment_timetable_recyclerview.visibility = View.VISIBLE
                 fragment_timetable_message.visibility = View.GONE
 
+                var _n = 0  // increases with every element
                 val items = mutableListOf<IFlexible<*>>()
-                val days_n = mutableListOf<Int>()
-                var current_n = 0
+                val today_day = Calendar.getInstance().get(Calendar.DAY_OF_WEEK)
+                var today_header = 0
 
                 for ((date, schoolDay) in timetableData) {
-                    days_n.add(current_n)
                     val header = LessonHeaderItem(date)
-                    current_n += 1
+                    val c = Calendar.getInstance()
```

Comment:
Please don't use old Java calendars. Use Joda instead. You got already prepared LocalDate date here